.github/README.md

```diff
@@ -19,7 +19,7 @@ Tabbycat is a draw tabulation system for British Parliamentary and a variety of
 
 ## 🔍 Features
 
-- Deployable to [Heroku](https://www.heroku.com/) for an easy, fast, and free setup
+- A range of deployment options — Tabbycat will run on Calico (a paid hosted service), Render or Heroku (free hosted services), and local installations (natively or via Docker)
```

Comment:
Wouldn't class Render or Heroku as "free hosted services", but something like "self-deployment" as a differentiator is the deployment obligations, not just the price.

Reply:
Revised in 9db7cf1008b1f5aa730e4d76135f045967e31dbd, let me know if you'd like any further edits.